LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2019 sugarbrain
```

Comment:
Is the first letter in sugarbrain in lowercase?

Reply:
done